platform/android/java/editor/build.gradle

```diff
@@ -43,9 +43,10 @@ android {
         }
```

Comment:
You may want to add `applicationIdSuffix ".debug"` to the debug build type so it can be installed side by side with the release build type.

@akien-mga Thoughts?